packages/interface-compliance-tests/src/connection-encryption/index.ts

```diff
@@ -47,8 +54,8 @@ export default (common: TestSetup<ConnectionEncrypter>): void => {
         inboundResult,
         outboundResult
       ] = await Promise.all([
-        crypto.secureInbound(remotePeer, localConn),
-        crypto.secureOutbound(localPeer, remoteConn, remotePeer)
+        cryptoRemote.secureInbound(localConn),
+        crypto.secureOutbound(remoteConn, remotePeer)
```

Comment:
similar question about dissimilarities between encrypters we create&call for `secureInbound` and `secureOutbound` calls